src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSourceProvider.cs

```diff
@@ -406,6 +469,13 @@ private SuggestedActionSet OrganizeRefactorings(Workspace workspace, CodeRefacto
                 }
 
                 var selectedSpan = selectedSpans[0];
+
+                // We only support refactorings when selected span intersects with the span that the light bulb is asking for.
+                if (!selectedSpan.IntersectsWith(range))
```

Comment:
@heejaechang are you implying that the selected span that we get by inspecting textview may from a different snapshot than the one that light bulb gave us? I'll add the following check for this case. Sounds good?

```if (selectedSpan.Snapshot.Version.ReiteratedVersionNumber != range.Snapshot.Version.ReiteratedVersionNumber)
{
    return null;
}```